README.md

```diff
@@ -1986,16 +1987,35 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+## Line length
+
+  - [26.1](#26.1) <a name='26.1'></a> Avoid very long lines of code longer than 100 characters.
+
+  > Why? This ensures readability and maintainability.
+
+    ```javascript
+    // bad
+    var foo = { bar: "This is a bar.", baz: { "qux": "This is a qux" }, difficult: "to read" };
+
+    // good
+    var foo = {
+      bar: "This is a bar.",
+      baz: {
+        qux: "This is a qux"
+      },
+      difficult: "to read"
+    };
+    ```
 
 ## ECMAScript 5 Compatibility
 
-  - [26.1](#26.1) <a name='26.1'></a> Refer to [Kangax](https://twitter.com/kangax/)'s ES5 [compatibility table](http://kangax.github.com/es5-compat-table/).
+  - [27.1](#27.1) <a name='26.1'></a> Refer to [Kangax](https://twitter.com/kangax/)'s ES5 [compatibility table](http://kangax.github.com/es5-compat-table/).
 
 **[⬆ back to top](#table-of-contents)**
 
 ## ECMAScript 6 Styles
 
-  - [27.1](#27.1) <a name='27.1'></a> This is a collection of links to the various es6 features.
```

Comment:
Named sections++

from the [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#instruct-the-structure-of-this-document):

> A rule reference number - e.g., C.7 (the 7th rule related to classes). Since the major sections are not inherently ordered, we use a letter as the first part of a rule reference "number". We leave gaps in the numbering to minimize "disruption" when we add or remove rules.